My homework is done

GalinaRubo opened the pull request:
I ' d got the same mistakes, as in class. The differens's directory Creational_design_patterns.git was  created

GalinaRubo committed:
Add files via upload

anst-foto reviewed (changes requested):
Жду выполненную работу

GalinaRubo committed:
Add files via upload

Работа с документами

GalinaRubo committed:
Update .gitignore

not used

anst-foto commented on DocManager.cpp:
```diff
@@ -0,0 +1,68 @@
+
+	Help();
+	cout << endl;
+	int com = 0;
```
Из названия не понятно что это за переменная

anst-foto reviewed (changes requested):
Зачем вставлять в решение код, который не нужен сейчас? Мне важно видеть иерархию классов, а не проверку правильности пути!

GalinaRubo committed:
Update DocManager.cpp

Just add commit to value "com"

GalinaRubo committed:
Update Manager.h

File are contaned only methods of working with a document

GalinaRubo committed:
Add files via upload

Add file, contaned class "Document". There are cteating path to the file of document, checing path, changing path depending upon methods

GalinaRubo committed:
Update Document.h

change encoding to UTF-8

anst-foto reviewed (commented):
Попробуйте сделать с использованием делегатов
